src/datasets/utils/info_utils.py

```diff
@@ -81,3 +82,10 @@ def get_size_checksum_dict(path: str) -> dict:
         for chunk in iter(lambda: f.read(1 << 20), b""):
             m.update(chunk)
     return {"num_bytes": os.path.getsize(path), "checksum": m.hexdigest()}
+
+
+def is_small_dataset(dataset_size):
```

Comment:
Maybe make it clear that it requires a value in bytes (not bits) ? In the docstring maybe ?